shared/src/business/utilities/documentGenerators/order.test.js

```diff
@@ -0,0 +1,149 @@
+const fs = require('fs');
+const path = require('path');
+const sass = require('sass');
+const {
+  applicationContext,
+} = require('../../test/createTestApplicationContext');
+const {
+  generatePdfFromHtmlInteractor,
+} = require('../../useCases/generatePdfFromHtmlInteractor');
+const { getChromiumBrowser } = require('../getChromiumBrowser');
+const { order } = require('./order');
+
+describe('documentGenerators', () => {
+  const testOutputPath = path.resolve(
+    __dirname,
+    '../../../../test-output/document-generation',
+  );
+
+  const writePdfFile = (name, data) => {
+    const pdfPath = `${testOutputPath}/${name}.pdf`;
+    fs.writeFileSync(pdfPath, data);
+  };
+
+  beforeAll(() => {
+    if (process.env.PDF_OUTPUT) {
+      fs.mkdirSync(testOutputPath, { recursive: true }, err => {
+        if (err) throw err;
+      });
+
+      applicationContext.getChromiumBrowser.mockImplementation(
+        getChromiumBrowser,
+      );
+
+      applicationContext.getNodeSass.mockImplementation(() => {
+        return sass;
+      });
+
+      applicationContext.getPug.mockImplementation(() => {
+        return require('pug');
+      });
+
+      applicationContext
+        .getUseCases()
+        .generatePdfFromHtmlInteractor.mockImplementation(
+          generatePdfFromHtmlInteractor,
+        );
```

Comment:
just noting that i'm seeing this repeated a bunch of times in these tests; might be a way to reuse it in `createTestApplicationContext`. AOK for Trello backlog if you agree.

Reply:
Good call!

Reply:
PRs incoming https://github.com/flexion/ef-cms/pull/8584